### ISSUE TITLE
Button for clearing all locations except home location

### ISSUE THREAD

**Issue by RAPXT** [breezy-weather/breezy-weather#361]
### Describe your idea

TLDR: see title

When traveling, I often search manually for a location. As
IP-based location doesn't work to well for me and androids inbuilt location service tends to drain my phones battery fast. However while traveling, I'll stay for a day to two weeks at most at foreign location. When my vacation is over I don't really need the other locations anymore. Therefore a button to clear out all locations except home location would be nice.

Thank you for your superb work on Breezy Weather <3

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] My idea is not related to an existing feature described in the [Homepage](https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md) document.
- [X] I have updated the app to **[version 4.5.4](https://github.com/breezy-weather/breezy-weather/releases/latest)**.

**Comment by papjul** [comment]
I believe you refer to "resident location" when talking about "home location", which is used for a different purpose than what you describe since it only has a purpose when added together with a current location. I can’t see of any way to make it easy to understand that this feature can also be used to "clear all but resident locations". And what about current location? Should it be cleared as well? Why?

I don’t know how many places you visit in one trip, but I did the following test:
1) I added my location + 10 random locations
2) I started a chronometer
3) I swiped deleted all locations except my location
4) I stopped the chronometer

It took me 6 seconds. If you travel once a year, that means 6 seconds lost per year vs 1 sec with a single button (probably more if this button is hidden in a submenu since a single icon without text label on location list will lack affordance).

In the current state, I see more harm than good of adding this feature:
- doesn’t fit well in UX
- can lead to "misclicks"
- overloads the interface for a very narrow case
- good chance it doesn’t help saving time

I also wonder: if you travel to multiple places in one trip, once you’re done visiting the first place, don’t you delete the first place from location list to let the second place you’re now visiting move to top? This would make your idea an unused feature if used that way.
